modules/microshift-uninstalling-lvms-csi-snapshot.adoc

```diff
@@ -0,0 +1,63 @@
+// Module included in the following assemblies:
+//
+// * microshift_storage/microshift-storage-plugin-overview.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-uninstalling-lvms-csi-snapshot_{context}"]
+== Uninstalling the CSI driver and CSI snapshot implementation
+
+To uninstall the installed CSI driver and CSI snapshot implementation, use the following procedure.
+
+.Procedure
+
+. To uninstall the CSI driver implementation, run all the following commands:
+* To delete the `vg-manager`, run the following command:
++
+[source,terminal]
+----
+$ oc delete -n openshift-storage daemonset.apps/vm-manager
```

Comment:
@copejon is it "vg manager" or "vm manager"? line 14 and 18 have difference.

Reply:
`vg-manager` is the correct version

Reply:
done